src/main.rs

```diff
@@ -409,23 +395,11 @@ fn remote_server_options(matches: &ArgMatches) -> Result<(Option<Sources>, Optio
         }
     };
 
-    let docker_config = if let Some(docker_config_json_path) =
-        matches.get_one::<String>("docker-config-json-path")
-    {
-        Some(read_docker_config_json_file(Path::new(
-            docker_config_json_path,
-        ))?)
-    } else if let Some(user_dir) = UserDirs::new() {
-        let config_json_path = user_dir.home_dir().join(".docker").join("config.json");
-        if Path::exists(&config_json_path) {
-            Some(read_docker_config_json_file(&config_json_path)?)
-        } else {
-            None
-        }
-    } else {
-        None
-    };
-    Ok((sources, docker_config))
+    if let Some(docker_config_json_path) = matches.get_one::<String>("docker-config-json-path") {
+        env::set_var(DOCKER_CONFIG_ENV_VAR, docker_config_json_path);
```

Comment:
nit, add a comment explaining why we are messing around with the env variable

Reply:
https://github.com/kubewarden/kwctl/pull/277/commits/b3c66376dba2c5b7bcde8ad396225b2fb384ea21